rb/.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.6
```

Comment:
@titusfortner We can also have it in the root folder, does it matter for RubyMine?